src/test/java/software/amazon/documentdb/jdbc/query/limitations/DocumentDbSqlInjectionTest.java

```diff
@@ -0,0 +1,301 @@
+/*
+ * Copyright <2021> Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ *
+ */
+
+package software.amazon.documentdb.jdbc.query.limitations;
+
+import org.bson.BsonArray;
+import org.bson.BsonDocument;
+import org.bson.BsonValue;
+import org.bson.conversions.Bson;
+import org.checkerframework.checker.nullness.qual.NonNull;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import software.amazon.documentdb.jdbc.common.test.DocumentDbFlapDoodleExtension;
+import software.amazon.documentdb.jdbc.query.DocumentDbMqlQueryContext;
+import software.amazon.documentdb.jdbc.query.DocumentDbQueryMappingService;
+import software.amazon.documentdb.jdbc.query.DocumentDbQueryMappingServiceTest;
+
+import java.sql.SQLException;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+
+import static software.amazon.documentdb.jdbc.calcite.adapter.DocumentDbRules.quote;
+
+@ExtendWith(DocumentDbFlapDoodleExtension.class)
+class DocumentDbSqlInjectionTest extends DocumentDbQueryMappingServiceTest {
+    private static final String COLLECTION_NAME = "testCollectionInjectionTest";
+    private static final String OTHER_COLLECTION_NAME = "otherTestCollectionInjectionTest";
+    private DocumentDbQueryMappingService queryMapper;
+
+    @BeforeAll
+    void beforeAll() throws SQLException {
+        final BsonDocument document =
+                BsonDocument.parse(
+                        "{ \"_id\" : \"key\", \"array\" : [ "
+                                + "{ \"field\" : 1, \"field1\": \"value\" }, "
+                                + "{ \"field\" : 2, \"field2\" : \"value\" , \"field3\" : { \"field4\": 3} } ]}");
+        final BsonDocument otherDocument =
+                BsonDocument.parse(
+                        "{ \"_id\" : \"key1\", \"otherArray\" : [ { \"field\" : 1, \"field3\": \"value\" }, { \"field\" : 2, \"field3\" : \"value\" } ]}");
```

Comment:
nit - formatting issue?

Reply:
It was copy/pasted - but formatted a more compact version.